packages/camera/camera_android_camerax/test/android_camera_camerax_test.dart

```diff
@@ -2,28 +2,42 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
+
+import 'package:async/async.dart';
 import 'package:camera_android_camerax/camera_android_camerax.dart';
+import 'package:camera_android_camerax/src/camera.dart';
 import 'package:camera_android_camerax/src/camera_info.dart';
 import 'package:camera_android_camerax/src/camera_selector.dart';
+import 'package:camera_android_camerax/src/camerax_library.g.dart';
+import 'package:camera_android_camerax/src/preview.dart';
 import 'package:camera_android_camerax/src/process_camera_provider.dart';
+import 'package:camera_android_camerax/src/system_services.dart';
+import 'package:camera_android_camerax/src/use_case.dart';
 import 'package:camera_platform_interface/camera_platform_interface.dart';
+import 'package:flutter/services.dart' show DeviceOrientation;
+import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:mockito/annotations.dart';
 import 'package:mockito/mockito.dart';
 
 import 'android_camera_camerax_test.mocks.dart';
 
-@GenerateMocks(<Type>[
-  ProcessCameraProvider,
-  CameraSelector,
-  CameraInfo,
+@GenerateNiceMocks(<MockSpec<Object>>[
```

Comment:
@stuartmorgan @bparrishMines Generate mocks for `BuildContext` this way has caused me to run into this error: https://cirrus-ci.com/task/6299387630977024?logs=unit_test#L203. This passes for me locally, so I'm wondering if y'all have any insight as to what might be happening with the CI?

Reply:
It looks like the error is about a typedef that was [added on master recently](https://github.com/flutter/flutter/pull/116429/files#diff-134bc36bd7c045a92ffff29f7efdaa7d375384911475a40343814c76c87b17cbR2028), so it's not on stable. You can probably just generate the mocks on `stable`.